pkg/compute/kube/fake/fake.go

```diff
@@ -95,3 +96,32 @@ func (f fakeClient) GetResolvableEndpointsForService(svc service.MeshService) []
 	}
 	return endpoints
 }
+
+func (f fakeClient) IsMetricsEnabled(*envoy.Proxy) (bool, error) {
+	return true, nil
+}
+
+// GetHostnamesForService returns the hostnames over which the service is accessible
+func (f fakeClient) GetHostnamesForService(svc service.MeshService, localNamespace bool) []string {
+	var hostnames []string
```

Comment:
seems like the fake has the same implementation as the actual one. Can this be consolidated?

Reply:
done